package.json

```diff
@@ -31,6 +31,7 @@
     "react-apollo": "^2.1.4",
     "react-dom": "^16.8.6",
     "react-highlight-words": "^0.16.0",
+    "react-hls-player": "^3.0.1",
```

Comment:
This should also be part of the second commit, since the `ReactHlsPlayer` is only used in the next commit with the front-end changes